Casks/jalview.rb

```diff
@@ -1,8 +1,8 @@
 cask 'jalview' do
-  version '2.11.0'
-  sha256 '640c43b9e564092d4f765b5b30bc901003a7ee80a49442544159bc0bfb207cfc'
+  version '2.11.1.0'
+  sha256 'd2bd8b4db3963381ace200c185e0ea4ebd63f61878a3466ecdc0cb1095d198c8'
 
-  url "https://www.jalview.org/getdown/release/install4j/1.8/Jalview-OFFLINE_macos-app_#{version.dots_to_underscores}-j8.dmg"
+  url "https://www.jalview.org/getdown/release/install4j/1.8/Jalview-#{version.dots_to_underscores}-macos-java_8.dmg"
   appcast 'https://www.jalview.org/getdown/release/'
```

Comment:
I think we might have a better candidate here:
```suggestion
  appcast 'https://www.jalview.org/getdown/release/install4j/1.8/updates.xml'
```